### ISSUE TITLE
COMFY ERROR / FLUX INPAINT: ClownsharKSampler_Beta 'attention_mask_img_shape'

### ISSUE THREAD

**Issue by pbries-glitch** [ClownsharkBatwing/RES4LYF#131]
# ComfyUI Error Report
## Error Details
- **Node ID:** 1053
- **Node Type:** ClownsharKSampler_Beta
- **Exception Type:** KeyError
- **Exception Message:** 'attention_mask_img_shape'

**Comment by pbries-glitch** [comment]
okay i have no issues with regional chroma though, which emans its something to do with flux related workflows

**Comment by pbries-glitch** [comment]
i think the error is orginating from model_base.py:

Around line 870?

    def extra_conds(self, **kwargs):
        out = super().extra_conds(**kwargs)
        cross_attn = kwargs.get("cross_attn", None)
        if cross_attn is not None:
            out['c_crossattn'] = comfy.conds.CONDRegular(cross_attn)
        # upscale the attention mask, since now we
        attention_mask = kwargs.get("attention_mask", None)
        if attention_mask is not None:
            shape = kwargs["noise"].shape
            mask_ref_size = kwargs["attention_mask_img_shape"]
            # the model will pad to the patch size, and then divide
            # essentially dividing and rounding up
            (h_tok, w_tok) = (math.ceil(shape[2] / self.diffusion_model.patch_size), math.ceil(shape[3] / self.diffusion_model.patch_size))
            attention_mask = utils.upscale_dit_mask(attention_mask, mask_ref_size, (h_tok, w_tok))
            out['attention_mask'] = comfy.conds.CONDRegular(attention_mask)
